sites/baseline/lists/_lists.scss

```diff
@@ -1,79 +0,0 @@
-/*
- *
- * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
- * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
- *
- * @title: Lists
- *
- */
-
-/*
- * List responsive style
- *
- * It has variant defined for some specific views
- * - views/_screen-lg-min
- * - views/_screen-xs-max
- */
-.list-responsive {
-
-	> li {
-		float: left;
-		padding-right: 5px;
-		width: 50%;
-
-		&:nth-child(2n+2) {
-			clear:right;
-		}
-	}
-
-	&:before, &:after {
-		content: " ";
-		display: table;
-	}
-	&:after {
-		clear: both;
-	}
-}
-
-/* Double space list items
-	Might worth to publish it upstream here instead: https://github.com/wet-boew/wet-boew/blob/master/src/base/lists/_base.scss#L30
- */
-
-.lst-spcd-2 {
-	> {
-		li {
-			margin-bottom: 20px;
-		}
-	}
-
-	ul,
-	ol {
-		margin-top: 20px;
-	}
```

Comment:
Please review the re-implementation of this style because it does interfere with the existing pattern.

See the list-fr.html for example

Reply:
Fixed. Thanks